app/uk/gov/hmrc/helptosavefrontend/models/IvResponse.scala

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2017 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.helptosavefrontend.models
+
+import uk.gov.hmrc.play.http.HttpResponse
+
+trait IvResponse
+
+case class IvSuccessResponse(result: String) extends IvResponse
+
+object IvSuccessResponse {
+  val Success = "Success"
```

Comment:
If there are only a limited number of success responses you can reflect this in the code by having 
```scala
sealed trait IvSuccessResponse
object IvSuccessResponse {
  case object Success extends IvSuccessResponse
  case object Incomplete extends IvSuccessResponse
  ...
  def fromString(s: String): Option[IvSuccessResponse] = ???
}

```